src/components/dashboard/DashboardPage.jsx

```diff
@@ -50,8 +51,7 @@ export default function DashboardPage() {
       <Container size="lg" className="py-5">
         <Row>
           <CourseEnrollmentsContextProvider>
-            {/* TODO: figure out how we want to present upgrade errors, i.e. pass through to LMS */}
-            {/* <CourseEnrollmentFailedAlert /> */}
+            <CourseEnrollmentFailedAlert className="pt-0 pb-3" enrollmentSource={ENROLLMENT_SOURCE.DASHBOARD} />
```

Comment:
nit: would using margin utility classes make more sense if this is regarding spacing between UI elements vs. padding within the element itself? though basically has the same end result here, using margin might feel a bit cleaner.

Reply:
I agree these should be margins.